### PR TITLE
[TSC] Fix for the readthedocs

### PR DESCRIPTION
**Pull request by rabi-abdel**
so the landing page of the readthedocs proposed by @CsatariGergely will have a more complete TOC

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
-* [Reference Model](ref_model)
-* [Reference Architecture](ref_arch)
-* [Reference Implementation](ref_impl)
-* [Reference Conformance](ref_cert)
-* [Vendor Implementations](ven_impl)
+* [Governance & Community Guidelines](gov)
+  * [Term of Reference](../GSMA_CNTT_Terms_of_Reference.pdf)
+  * [Code of Conduct](../CODE_OF_CONDUCT.md)
+  * [The License (Creative Commons Attribution 4.0 International)](https://creativecommons.org/licenses/by/4.0/legalcode)
+* [Technical Specefications](tech)
```
Comment (tomkivlin) [inline doc/README.md:8]:
```suggestion
* [Technical Specifications](tech)
```

File: doc/README.md [hunk 2]
```diff
@@ -1,7 +1,14 @@
...
+
 ## Table of Contents
-* [Community Guidelines](gov)
-* [Reference Model](ref_model)
-* [Reference Architecture](ref_arch)
-* [Reference Implementation](ref_impl)
-* [Reference Conformance](ref_cert)
-* [Vendor Implementations](ven_impl)
+* [Governance & Community Guidelines](gov)
+  * [Term of Reference](../GSMA_CNTT_Terms_of_Reference.pdf)
```
Comment (tomkivlin) [inline doc/README.md:5]:
```suggestion
  * [Terms of Reference](../GSMA_CNTT_Terms_of_Reference.pdf)
```

File: doc/ref_cert/README.md [hunk 3]
```diff
@@ -132,5 +132,5 @@ the network settings:
 
 <a name="available-cr"></a>
 ## Available Programs
-* [RC1 - LFN Based](lfn)
+* [RC1 - Openstack Based](lfn)
```
Comment (pgoyal01) [inline doc/ref_cert/README.md:135]: Is it possible to rename the Github "lfn" directory to "RC1" so as to be consistent?

Comment (rabi-abdel) [reply doc/ref_cert/README.md:135]: OK, will do that.

Comment (rabi-abdel) [reply doc/ref_cert/README.md]: fixed in https://github.com/cntt-n/CNTT/pull/1757/commits/3c622a3900b971577c30a01534848fef798e3815

**Comment by rabi-abdel** [comment]
> minor nitpicks

@tomkivlin all fixed

File: doc/ref_cert/README.md (hunk 3 above)
Comment (pgoyal01) [reply doc/ref_cert/README.md]: @rabi-abdel Thanks.

**Comment by ASawwaf** [comment]
thx @CsatariGergely

**Pull request merged**